condor_watch_q.py

```diff
@@ -678,9 +693,15 @@ def ordered(cls):
     htcondor.JobEventType.JOB_SUSPENDED: JobStatus.SUSPENDED,
     htcondor.JobEventType.JOB_ABORTED: JobStatus.REMOVED,
 }
-
-
-def table(headers, rows, fill="", header_fmt=None, row_fmt=None, alignment=None):
+class color:
+    RED = '\033[31m'    #HOLD: needs intervention      
+    GREEN = '\033[32m'  #COMPLETE: complete without issues
+    CYAN = '\033[36m'   #RUNNING: working as intended, no hold
+    BLUE = '\033[34m'   #NOT STARTED
+    GREY = '\033[37m'   #unknown case
+    ENDC = '\033[0m'
```

Comment:
This looks like an enum; make it so for real: https://docs.python.org/3/library/enum.html . In particular, I think you want a "string enum", like https://github.com/htcondor/htmap/blob/master/htmap/utils.py#L39

The class name should be capitalized.

I would also get rid the comments; the colors don't necessarily have meaning by themselves. That meaning is granted by the function you defined above.